packages/react/src/components/Chip/Chip.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import type { Meta, StoryObj, StoryFn } from '@storybook/react';
+
+import { Chip } from './index';
+
+type Story = StoryObj<typeof Chip>;
+
+export default {
+  title: 'Kjernekomponenter/Chip',
+  component: Chip,
+  parameters: {
+    status: {
+      type: 'beta',
+      url: 'http://www.url.com/status',
+    },
+  },
+} as Meta;
+
+export const Preview: Story = {
+  args: {
+    children: 'Nynorsk',
+    size: 'small',
+    selected: false,
+    checkmark: false,
+  },
+};
+
+export const Default: StoryFn<typeof Chip> = () => {
+  return <Chip>Nynorsk</Chip>;
+};
+
+export const ChipWithCheckmark: StoryFn<typeof Chip> = () => {
+  return (
+    <Chip
+      checkmark
```

Comment:
Hvorfor har ikke ChipWithCheckmark onClick prop?

Reply:
Lurte litt på det samme egentlig.
Hvordan skal man kunne endre fra unselected til selected hvis den ikke har noen onClick?

Eller er det ikke meningen at dette skal være en intraktiv knapp utover når den er deletable?

Reply:
For nå blir den kun blå (og evt. checked) når den har fokus..
Er det meningen?

Reply:
Whether this should be more visible in storybook is a different matter - has it been discussed?